src/app/Caches/Create/AdvancedOptions.tsx

```diff
@@ -0,0 +1,183 @@
+            isolationLevel: isolationLevel,
+            lockAcquisitionTimeout: lockAcquisitionTimeout,
+            striping: striping,
+        });
+    }, [storage, concurrencyLevel, isolationLevel, lockAcquisitionTimeout, striping]);
+
+    const onToggleIsolationLevel = () => {
+        setIsOpenIsolationLevel(!isOpenIsolationLevel);
+    };
+
+    const onSelectIsolationLevel = (event, selection, isPlaceholder) => {
+        setIsolationLevel(selection);
+        setIsOpenIsolationLevel(false);
+    };
+
+    const handleConcurrencyLevel = (value) => {
+        setConcurrencyLevel(value);
+    }
+
+    const handleLockAcquisitionTimeout = (value) => {
+        setLockAcquisitionTimeout(value);
+    }
+
+    const formMemory = () => {
+        return (
+            <React.Fragment>
+                <TextContent>
+                    <Text component={TextVariants.h3}>{t('caches.create.configurations.advanced-options.page-title')}</Text>
+                </TextContent>
+                <TextContent>
+                    <Text component={TextVariants.p}>{t('caches.create.configurations.advanced-options.page-subtitle')}</Text>
+                </TextContent>
+                <FormGroup
+                    isInline
+                    isRequired
+                    fieldId="field-storage"
+                >
+                    <MoreInfoTooltip label={t('caches.create.configurations.advanced-options.storage-title')} toolTip={t('caches.create.configurations.advanced-options.storage-tooltip')} textComponent={TextVariants.h4} />
+                    <Radio
+                        name="radio"
+                        id="heap"
+                        onChange={() => setStorage(StorageType.HEAP)}
+                        isChecked={storage as StorageType == StorageType.HEAP}
+                        label={
+                            <TextContent>
+                                <Text component={TextVariants.h3}>
+                                    {t('caches.create.configurations.advanced-options.radio-heap')}
+                                </Text>
+                            </TextContent>
+                        }
+                    />
+                    <Radio
+                        name="radio"
+                        id="off_heap"
+                        onChange={() => setStorage(StorageType.OFF_HEAP)}
+                        isChecked={storage as StorageType == StorageType.OFF_HEAP}
+                        label={
+                            <TextContent>
+                                <Text component={TextVariants.h3}>
+                                    {t('caches.create.configurations.advanced-options.radio-off-heap')}
+                                </Text>
+                            </TextContent>
+                        }
+                    />
+                </FormGroup>
+            </React.Fragment>
+        );
+    }
+
+    // Options for Isolation Level
+    const isolationLevelOptions = () => {
```

Comment:
@dipguptaredhat and @karesti I think the console should show only the supported options.

Need to remove/hide "SERIALIZABLE" and "READ_UNCOMMITTED" from the drop-down. Should we do that in this PR or open a separate issue?

@pruivo FYI can we remove these options from the schema as well?

Reply:
Actually there should only be two options for the isolation level: READ_COMMITTED and REPEATABLE_READ. NONE makes no sense for clustered caches.

Reply:
@oraNod TBH, I want to remove the `isolation-level` attribute :rofl:
See point 2 in https://issues.redhat.com/browse/ISPN-3927
+1 to remove the unsupported options